tmt/steps/provision/artemis.py

```diff
@@ -548,7 +548,22 @@ def _create(self) -> None:
         # TODO: spot instance
 
         if self.post_install_script:
-            data['post_install_script'] = self.post_install_script
+            if tmt.utils.is_url(self.post_install_script):
+                try:
+                    response = requests.get(self.post_install_script)
+                    response.raise_for_status()
+
+                except requests.exceptions.RequestException:
+                    raise ArtemisProvisionError(
+                        "failed to get the post_install_script file")
+
+                post_install_script = response.text
+            else:
+                self.info('post-install-script argument is treated as a raw script')
```

Comment:
Is this worth of 'info' level? Isn't it more like debug?

Reply:
@lukaszachy Thanks for your review,updated accordingly:)